src/formatDate.js

```diff
@@ -8,7 +8,89 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let result = '';
+
+  let symbol = '';
+
+  for (let i = 0; i < date.length; i++) {
+    if (isNaN(date[i])) {
+      symbol = date[i];
+    }
+  }
+
+  const slicedDate = date.split(symbol);
+
+  const indexOfDay = fromFormat.indexOf('DD');
+  const indexOfMonth = fromFormat.indexOf('MM');
+  const indexOfYear = findindexOfYear(fromFormat, slicedDate);
+
+  const day = slicedDate[indexOfDay];
+  const month = slicedDate[indexOfMonth];
+  let year = slicedDate[indexOfYear];
+
+  const toIndexOfDay = toFormat.indexOf('DD');
+  const toIndexOfMonth = toFormat.indexOf('MM');
+  const toIndexOfYear = toFindindexOfYear(toFormat);
```

Comment:
The function `toFindindexOfYear` is not defined before its usage. Make sure to define the function before calling it to avoid reference errors.